clang-tools-extra/clangd/test/implicit-function.test

```diff
@@ -0,0 +1,11 @@
+// RUN: cp %s %t.c
```

Comment:
I think it would be more consistent with the rest of the clangd subproject if this was expressed as a unit test.

It can take the form of a new section added to [this existing test](https://searchfox.org/llvm/rev/6a91b7051df7e108c8ffd79c084859f90559962d/clang-tools-extra/clangd/unittests/tweaks/ExtractVariableTests.cpp#19), similar to [this section](https://searchfox.org/llvm/rev/6a91b7051df7e108c8ffd79c084859f90559962d/clang-tools-extra/clangd/unittests/tweaks/ExtractVariableTests.cpp#61-66) but using `EXPECT_UNAVAILABLE` (since the refactoring does not end up being available on this test case, even though it passes this particular check). The `[[ ]]` in the input source code annotates the source range on which the refactoring is invoked.

Reply:
I've been trying to do that, but so far no luck. I hit this assertion in TweakTesting.h:
`      assert(!A.points().empty() || !A.ranges().empty()); 
`

Reply:
Well, I forgot to add the [[]] thing around the call. Now I'm back on track again :-)